docs/getting-started/integrations-cdn/next-js.md

```diff
@@ -17,8 +17,110 @@ CKEditor&nbsp;5 does not support server-side rendering yet, but you can integrat
 
 {@snippet getting-started/use-builder}
 
-You get ready-to-use code tailored to your needs! You can take the output from the builder, specifically the npm React snippet, and follow the npm path below. Just replace the content of the `components/custom-editor.js` file. The snippet may contain client-side hooks, so do not forget about adding the `'use client'` directive in the case of the App Router.
-
 ## Setting up the project
 
 This guide assumes you already have a Next project. To create such a project, you can use CLI like `create-next-app`. Refer to the [Next.js documentation](https://nextjs.org/docs/app/api-reference/create-next-app) to learn more.
+
+## Using from CDN
+
+First, install the CKEditor 5 packages:
+
+* `ckeditor5` &ndash; package with open-source plugins and features.
+* `ckeditor5-premium-features` &ndash; package with premium plugins and features.
+
+Depending on your configuration and selected plugins, you may need to install only the first one or both.
+
+```bash
+npm install ckeditor5 ckeditor5-premium-features
+```
+
+Next.js is based on React, so install the [CKEditor 5 WYSIWYG editor component for React](https://www.npmjs.com/package/@ckeditor/ckeditor5-react), too:
+
+```bash
+npm install @ckeditor/ckeditor5-react
+```
+
+You will use the installed dependencies in a React component. Create a new component in the components directory, for example, `components/custom-editor.js`. Inside the component file, import all necessary dependencies. Then, create a functional component that returns the CKEditor&nbsp;5 React component. The example below shows how to use the component with both open-source and premium plugins.
+
+The App Router, by default, uses server components. It means you need to mark a component as client-side explicitly. You can achieve that by using the `'use client'` directive at the top of the file, above your imports. You do not need the directive if you use the Pages Router.
+
+In the below example, the `useCKEditorCloud` hook is used to load the editor code and plugins from CDN. To use premium plugins, set the `premium` property to `true` and provide your license key in the configuration. For more information about the `loadCKEditorCloud` helper, see the {@link getting-started/setup/loading-cdn-resources Loading CDN resources} guide.
```

Comment:
DOn't we need to provide the licence call anyway, just without premium it's `GPL`? This might need to be clarified.